### DISCUSSION TITLE
Welcome to mmcif_pdbx Discussions!

### DISCUSSION THREAD

**Discussion by epeisach** [wwpdb-dictionaries/mmcif_pdbx#36] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by drlemmus** [comment]
Hi everyone, Robbie Joosten here,

My interest with mmCIF_pdbx is that I'm a developer of software for structure model analysis, validation and optimisation (e.g. PDB-REDO, DSSP and Tortoize). Quite a lot of things we work have been around for quite a while and we see the advantages and the challenges of moving from PDB as the main format of structure models to mmCIF. It makes it easier for use to capture the data that we want both as input and output. At the same time this is the moment to do things right and make sure we do our part to keep one standard that everyone can use. The best way to do that is to collaborate where the standard is defined and continues to be improved. And that's right here!
